### PR TITLE
Feature/frontend/views/app created

### PR DESCRIPTION
**Pull request by rem1niscence**

### CODE REVIEW

**Comment by xerp** [comment]
@justaname94 The CI lint inform that code has errors.

File: frontend/src/_routes.js [hunk 1]
```diff
@@ -60,6 +64,18 @@ export const dashboardRoutes = [
     exact: true,
     name: "Home - Dashboard",
     component: Dashboard,
+  },
+  {
+    path: dashboardPaths.appCreated,
+    exact: true,
+    name: "Your app's been created",
```
Comment (xerp) [inline frontend/src/_routes.js]: Please, change the name of this value.

File: frontend/src/views/Dashboard/Apps/AppsMain/AppCreated/AppCreated.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+import "./AppCreated.scss";
+import InfoCard from "../../../../../core/components/InfoCard/InfoCard";
+import {PropTypes} from "prop-types";
+
+// TODO: Move constants to constants file.
+const BONDSTATUS = {
+  0: "Bonded",
+  1: "Unbonding",
+  2: "Unbonded",
+};
```
Comment (xerp) [inline frontend/src/views/Dashboard/Apps/AppsMain/AppCreated/AppCreated.js]: This code is duplicated on another file.

**Review by xerp — CHANGES_REQUESTED** [review]
There are pending changes.

File: frontend/src/_routes.js (hunk 1 above)
Comment (rem1niscence) [reply frontend/src/_routes.js]: which one of the three?

File: frontend/src/_routes.js [hunk 3]
```diff
@@ -68,7 +68,7 @@ export const dashboardRoutes = [
   {
     path: dashboardPaths.appCreated,
     exact: true,
-    name: "Your app's been created",
+    name: "App created",
```
Comment (xerp) [inline frontend/src/_routes.js]: This is the name of the route, no for actions. So this name could be App creation or something else.

**Review by xerp — CHANGES_REQUESTED** [review]
There are pending changes

**Review by xerp — APPROVED** [review]
Looks good

**Review by xerp — APPROVED** [review]
Looks good

**Pull request merged**